geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/StatParser.scala

```diff
@@ -102,6 +102,13 @@ private class StatParser extends BasicParser {
     }
   }
 
+  private def stats: Rule1[Stat] = rule {
+    "Stats(" ~ string ~ ")" ~~> { attributes =>
```

Comment:
Sure, +1 on the name change.  (To be fair to Tom, I'm the one who wired this up initially, so the poor name choice is all me;).)